DeAceroDLabs/marketplace-frontend
# fix: [DMP-613] Refactor Button

## What is done

[[DMP-613] Refactor del component de Button](https://deacero.atlassian.net/browse/DMP-613)

## How is done

1. Refactor Button component.
2. Updated possible colors prop.
3. Added border prop

## How to test

1. Run the project via `npm start`
4. Check that all the buttons in the project work and look as previous.

**NOTE: Some buttons changed the font size the had before. I think is better this way so it feels more natural across the app (You will barely notice)**
